content/en/docs/setup/install/helm/index.md

```diff
@@ -68,10 +63,9 @@ instead with the flags `--set gateways.istio-ingressgateway.type=NodePort`
 appended to the end of the Helm instructions in the installation steps below.
 {{< /tip >}}
 
-### Option 1: Install with Helm via `helm template`
-
-Choose this option if your cluster doesn't have [Tiller](https://helm.sh/docs/topics/architecture/#components)
-deployed and you don't want to install it.
+{{< tip >}}
+The Helm installation method previously detailed here that used the [Tiller](https://helm.sh/docs/topics/architecture/#components) component has been removed. The preferred install path is to use `istioctl` as documented in [Installing with {{< istioctl >}}](/docs/setup/install/istioctl/). If you still desire to use Helm, you'll need to leverage the `helm template` method detailed below.
```

Comment:
```suggestion
This document used to also describe a Helm installation method that used the [Tiller](https://helm.sh/docs/topics/architecture/#components) component. That installation method is no longer available. Instead, we recommend using `istioctl`, as documented in [Installing with {{< istioctl >}}](/docs/setup/install/istioctl/). If you want to use Helm, then you need to use the `helm template` method described below.
```

...Or something like that. My concerns here are:

* The phrase "The Helm installation method previously detailed here" is slightly hard to parse. But only slightly--if you want to keep that phrase as-is, that's fine with me.

* The words "desire" and "leverage" are fancier words than are needed here--I recommend simplifying the language for ease of reading.